templates/xcassets/swift3.stencil

```diff
@@ -10,7 +10,7 @@
 {% set imageAlias %}{{param.imageAliasName|default:"AssetImageTypeAlias"}}{% endset %}
 {% set forceNamespaces %}{{param.forceProvidesNamespaces|default:"false"}}{% endset %}
 {% set accessModifier %}{% if param.publicAccess %}public{% else %}internal{% endif %}{% endset %}
-#if os(OSX)
+#if os(macOS)
```

Comment:
So did you check that the name macOS was available as soon as 3.0 and not just 3.1 then (as Brent wasn't totally sure in Slack)?

Reply:
Jep, it's in the 3.0 branch:
https://github.com/apple/swift/blob/swift-3.0-branch/lib/AST/PlatformKind.cpp#L54

Reply:
And the 3.0 release: https://github.com/apple/swift/blob/swift-3.0-RELEASE/lib/AST/PlatformKind.cpp